tools/src/test/java/org/apache/kafka/tools/GetOffsetShellTest.java

```diff
@@ -333,7 +382,7 @@ private void assertExitCodeIsOne(String... args) {
     }
 
     private List<Row> expectedOffsetsWithInternal() {
-        List<Row> consOffsets = IntStream.range(0, offsetTopicPartitionCount + 1)
+        List<Row> consOffsets = IntStream.range(0, offsetTopicPartitionCount)
```

Comment:
This is a side fix for this test. Before this PR, the `offsetTopicPartitionCount` doesn't feed into cluster.